tests/integration/security/policy_attachment_only/jwt_gateway_test.go

```diff
@@ -190,6 +190,98 @@ func TestGatewayAPIRequestAuthentication(t *testing.T) {
 		})
 }
 
+func TestGatewayAPIAuthorizationPolicy(t *testing.T) {
+	framework.NewTest(t).
+		Label(label.IPv4). // https://github.com/istio/istio/issues/35835
+		Features("security.authorization.policy-attachment").
+		Run(func(t framework.TestContext) {
+			crd.DeployGatewayAPIOrSkip(t)
+			config.New(t).
+				Source(config.File("testdata/authz/gateway-api.yaml.tmpl").WithParams(param.Params{
+					param.Namespace.String(): apps.Namespace,
+				})).
+				Source(config.File("testdata/authz/gateway-authz.yaml.tmpl").WithParams(param.Params{
+					param.Namespace.String(): apps.Namespace,
+					"Services":               apps.A.Append(apps.B).Services(),
+				})).
+				BuildAll(nil, apps.A.Append(apps.B).Services()).
+				Apply()
+
+			t.NewSubTest("gateway-authz-policy-attachment-only").Run(func(t framework.TestContext) {
+				test.SetForTest(t, &features.EnableSelectorBasedK8sGatewayPolicy, false)
+				cases := []struct {
+					name          string
+					customizeCall func(opts *echo.CallOptions, to echo.Target)
+				}{
+					{
+						name: "allow with sub-1 token",
+						customizeCall: func(opts *echo.CallOptions, to echo.Target) {
+							opts.HTTP.Path = "/allow"
+							opts.HTTP.Method = "GET"
+							opts.HTTP.Headers = headers.New().
+								WithHost(fmt.Sprintf("example.%s.com", to.ServiceName())).
+								WithAuthz(jwt.TokenIssuer1).
+								Build()
+							opts.Check = check.OK()
+						},
+					},
+					{
+						name: "deny without token",
```

Comment:
Do we need a test case for denying with an incorrect token?

Reply:
I think its good to have as it confirms the `AuthPolicy` is enforcing only requests with valid token. Happy to remove if others think its not necessary.

Reply:
I was curious if we wanted a test case with an incorrect token in addition to the test case with no token.

Reply:
Oh I misunderstood!